next/services/graphql/queries/common.graphql

```diff
@@ -17,12 +17,43 @@ fragment UploadFileEntity on UploadFileEntity {
 fragment UploadImageEntity on UploadFileEntity {
   id
   attributes {
-    url
-    name
-    alternativeText
-    caption
-    size
-    width
-    height
+    ...UploadImage
+  }
+}
+
+fragment UploadImage on UploadFile {
+  url
+  name
+  alternativeText
+  caption
+  size
+  width
+  height
+}
+
+fragment EventTags on EventTag {
+  title
+  slug
+  createdAt
+  publishedAt
+  updatedAt
+}
+
+fragment EventCategory on EventCategory {
+  title
+  createdAt
+  updatedAt
+}
+
+fragment Branch on Branch {
```

Comment:
Please name this EventBranch (or something like this) and take only the attributes you need for event card. Otherwise it looks like the main `Branch` type we should use everywhere :D